app/src/test/java/com/kickstarter/viewmodels/SignupViewModelTest.kt

```diff
@@ -105,7 +105,7 @@ class SignupViewModelTest : KSRobolectricTestCase() {
 
         formSubmittingTest.assertValues(true, false)
         signupSuccessTest.assertValueCount(1)
-        environment().currentUserV2()?.observable()?.subscribe {
+        environment.currentUserV2()?.observable()?.subscribe {
```

Comment:
Fixed a minor error in this class while using it as a reference to write tests for TwoFactorViewModelTest